cli/deno_dir.rs

```diff
@@ -351,7 +361,15 @@ impl SourceMapGetter for DenoDir {
   }
 
   fn get_source_line(&self, script_name: &str, line: usize) -> Option<String> {
-    match self.fetch_module_meta_data(script_name, ".", true, true) {
+    // TODO(bartlomieju): this is temporary hack to make tests pass
+    // TODO: this method shouldn't issue `fetch_module_meta_data` - this is done for each line
```

Comment:
Reference: https://github.com/denoland/deno/commit/425df50484f315dcd63b4d93ab6911702779899e#comments

Reply:
get_source_line() and get_source_map() seem to be very similar in structure. Is it possible to unify this code?